### PR TITLE
keystone/enhancement/issue-16-standarizeAPI

### PR DESCRIPTION
**Pull request by cxoltero**
kenzanlabs/keystone#7
fixes issue #16

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

**Comment by obuckley-kenzan** [comment]
@cxoltero per the CONTRIBUTING.MD, please reference the issue this resolves, using the `#`

**Comment by obuckley-kenzan** [comment]
ok to test

File: src/index.js [hunk 1]
```diff
@@ -17,31 +17,35 @@ var config = {
...
+    }
+
+    return minifyJS();
+  }
+};
 
-  var pipeline = {
-    minifyJS: minifyJS()
-  };
+function minifyJS() {
```
Comment (obuckley-kenzan) [inline src/index.js]: Could we just do `return makePipe();` here? Just curious the value in the additional function call / wrapper of `minifyJS`?

cc: @paul-barry-kenzan

Comment (paul-barry-kenzan) [inline src/index.js]: @cxoltero @obuckley-kenzan 
I have to agree with Owen here. The `makePipe()` should be sufficient.

**Comment by obuckley-kenzan** [comment]
@cxoltero 
Maybe i missed it, but why was this closed?

**Comment by cxoltero** [comment]
I submitted 2 PRs, one containing the code in this one. I believe...

**Pull request closed without merging**